File: js-TDD/fizz-buzz.js
function fizzBuzz(num) {
    if(parseInt(num)!= num){
        return "Invalid Input Data"
    }
    if (num % 15 === 0) {
        return "FIZZBUZZ";
    } else {
        if (num % 3 === 0) {
            return "FIZZ";
        }
        if (num % 5 === 0) {
            return "BUZZ";
        }
    }

    return;
}

module.exports = fizzBuzz;

File: TDD-java/src/FizzBuzz.java
/**
 * Created by gaole on 7/10/16.
 */
public class FizzBuzz {
    public String work(Object input) {
        int inputNum = (Integer) input;

        if (inputNum % 3 == 0) {
            return "FIZZ";
        }
        if (inputNum % 5 == 0) {
            return "BUZZ";
        }
        return "";
    }
}
